### PR TITLE
docs: changelog v5.2.1

### PR DESCRIPTION
**Pull request by vagusX**
- 🐞 修复 Silder 中的标记点 dot 可以正确触发 hover 和 click 的效果。[#40679](https://github.com/ant-design/ant-design/pull/40679) [@LongHaoo]
- 🐞 修复 Tour 不支持 `0` 作为节点的问题。[#40631](https://github.com/ant-design/ant-design/pull/40631) [@li-jia-nan](https://github.com/li-jia-nan)
- 💄 修复 DataPicker.RangePicker 的 hover 范围样式。[#40607](https://github.com/ant-design/ant-design/pull/40607) [@Yuiai01]
- 💄 优化 Steps 组件自定义 `icon` 的大小。[#40672](https://github.com/ant-design/ant-design/pull/40672) [@MadCcc](https://github.com/MadCcc)
- TypeScript
  - 🤖 类型：更新 Upload 组件以支持泛型。[#40634](https://github.com/ant-design/ant-design/pull/40634) [@riyadelberkawy](https://github.com/riyadelberkawy)
- 🌐 国际化
  - 🇷🇺/🇺🇦 补全 `ru_RU` 和 `uk_UA` 文案。[#40656](https://github.com/ant-design/ant-design/pull/40656) [@eldarcodes](https://github.com/eldarcodes)

-----

- 🐞 Fix Silder dot can trigger click and hover correctly. [#40679](https://github.com/ant-design/ant-design/pull/40679) [@LongHaoo]
- 🐞 Fix Tour that should support `0` as element. [#40631](https://github.com/ant-design/ant-design/pull/40631) [@li-jia-nan](https://github.com/li-jia-nan)
- 💄 Fix DataPicker.RangePicker hover range style. [#40607](https://github.com/ant-design/ant-design/pull/40607) [@Yuiai01]
- 💄 Optimize Steps custom `icon` size. [#40672](https://github.com/ant-design/ant-design/pull/40672) [@MadCcc](https://github.com/MadCcc)
- TypeScript
  - 🤖 Update Upload to support generic types. [#40634](https://github.com/ant-design/ant-design/pull/40634) [@riyadelberkawy](https://github.com/riyadelberkawy)
- 🌐 Localization
  - 🇷🇺/🇺🇦 add missing translations for ru_RU and uk_UA. [#40656](https://github.com/ant-design/ant-design/pull/40656) [@eldarcodes](https://github.com/eldarcodes)

### CODE REVIEW

**Comment by li-jia-nan** [comment]
@SocketSecurity ignore @swc/core@1.3.35
@SocketSecurity ignore core-js@3.27.1
@SocketSecurity ignore core-js@3.27.2
@SocketSecurity ignore core-js-pure@3.27.2
@SocketSecurity ignore esbuild@0.15.18

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,16 @@ timeline: true
...
 ---
 
+## 5.2.1
+
+`2023-2-13`
+
+- 💄 Optimize Steps custom `icon` size. [#40672](https://github.com/ant-design/ant-design/pull/40672) [@MadCcc](https://github.com/MadCcc)
+- TypeScript
+  - 🤖 Update Upload to support generic types. [#40634](https://github.com/ant-design/ant-design/pull/40634) [@riyadelberkawy](https://github.com/riyadelberkawy)
+- 🌐 国际化
```
Comment (li-jia-nan) [inline CHANGELOG.en-US.md:25]:
```suggestion
- 🌐 Localization
```

**Review by zombieJ — APPROVED** [review]

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/40631 呢

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/40607

**Comment by MadCcc** [comment]
- [ ] https://github.com/ant-design/ant-design/pull/40679

**Comment by yoyo837** [comment]
等下更新下tabs?

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,19 @@ timeline: true
...
+`2023-2-13`
+
+- 🐞 Fix Silder dot can trigger click and hover correctly. [#40679](https://github.com/ant-design/ant-design/pull/40679) [@LongHaoo](https://github.com/LongHaoo)
+- 🐞 Fix Tour that should support `0` as element. [#40631](https://github.com/ant-design/ant-design/pull/40631) [@li-jia-nan](https://github.com/li-jia-nan)
+- 💄 Fix DataPicker.RangePicker hover range style. [#40607](https://github.com/ant-design/ant-design/pull/40607) [@Yuiai01](https://github.com/Yuiai01)
+- 💄 Optimize Steps custom `icon` size. [#40672](https://github.com/ant-design/ant-design/pull/40672) [@MadCcc](https://github.com/MadCcc)
+- TypeScript
+  - 🤖 Update Upload to support generic types. [#40634](https://github.com/ant-design/ant-design/pull/40634) [@riyadelberkawy](https://github.com/riyadelberkawy)
+- 🌐 Localization
+  - 🇮🇷/🇺🇦 add missing translations for ru_RU and uk_UA. [#40656](https://github.com/ant-design/ant-design/pull/40656) [@eldarcodes](https://github.com/eldarcodes)
```
Comment (yoyo837) [inline CHANGELOG.en-US.md:29]: 棋子错了吧? 左边这个是伊朗?

Comment (yoyo837) [reply CHANGELOG.en-US.md:29]:
```suggestion
  - 🇷🇺/🇺🇦 add missing translations for ru_RU and uk_UA. [#40656](https://github.com/ant-design/ant-design/pull/40656) [@eldarcodes](https://github.com/eldarcodes)
```

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,19 @@ timeline: true
...
+`2023-2-13`
+
+- 🐞 修复 Silder 中的标记点 dot 可以正确触发 hover 和 click 的效果。[#40679](https://github.com/ant-design/ant-design/pull/40679) [@LongHaoo](https://github.com/LongHaoo)
+- 🐞 修复 Tour 不支持 `0` 作为节点的问题。[#40631](https://github.com/ant-design/ant-design/pull/40631) [@li-jia-nan](https://github.com/li-jia-nan)
+- 💄 修复 DataPicker.RangePicker 的 hover 范围样式。[#40607](https://github.com/ant-design/ant-design/pull/40607) [@Yuiai01](https://github.com/Yuiai01)
+- 💄 优化 Steps 组件自定义 `icon` 的大小。[#40672](https://github.com/ant-design/ant-design/pull/40672) [@MadCcc](https://github.com/MadCcc)
+- TypeScript
+  - 🤖 类型：更新 Upload 组件以支持泛型。[#40634](https://github.com/ant-design/ant-design/pull/40634) [@riyadelberkawy](https://github.com/riyadelberkawy)
+- 🌐 国际化
+  - 🇮🇷/🇺🇦 补全 `ru_RU` 和 `uk_UA` 文案。[#40656](https://github.com/ant-design/ant-design/pull/40656) [@eldarcodes](https://github.com/eldarcodes)
```
Comment (yoyo837) [inline CHANGELOG.zh-CN.md:29]:
```suggestion
  - 🇷🇺/🇺🇦 补全 `ru_RU` 和 `uk_UA` 文案。[#40656](https://github.com/ant-design/ant-design/pull/40656) [@eldarcodes](https://github.com/eldarcodes)
```

File: CHANGELOG.en-US.md (hunk 2 above)
Comment (li-jia-nan) [reply CHANGELOG.en-US.md:29]: 这都被你发现了，太细了

**Comment by li-jia-nan** [comment]
> #40670

这个是内部方法，用户应该是无感知的吧，除非他自己用了我们的内部方法

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,21 @@ timeline: true
 
 ---
 
+## 5.2.1
+
+`2023-2-13`
+
+- 🛠 重构 Tour 中 `panelRender` 函数为 FC。[#40670](https://github.com/ant-design/ant-design/pull/40670) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🐞 Fix `className` property wrongly passed to child nodes in TimeLine。[#40700](https://github.com/ant-design/ant-design/pull/40700) [@any1024](https://github.com/any1024)
+- 🐞 修复 Silder 中的标记点 dot 可以正确触发 hover 和 click 的效果。[#40679](https://github.com/ant-design/ant-design/pull/40679) [@LongHaoo](https://github.com/LongHaoo)
```
Comment (MadCcc) [inline CHANGELOG.zh-CN.md:24]:
```suggestion
- 🐞 修复 Silder 中的标记点在边缘无法点击的问题。[#40679](https://github.com/ant-design/ant-design/pull/40679) [@LongHaoo](https://github.com/LongHaoo)
```

File: CHANGELOG.zh-CN.md [hunk 5]
```diff
@@ -15,6 +15,21 @@ timeline: true
 
 ---
 
+## 5.2.1
+
+`2023-2-13`
+
+- 🛠 重构 Tour 中 `panelRender` 函数为 FC。[#40670](https://github.com/ant-design/ant-design/pull/40670) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🐞 Fix `className` property wrongly passed to child nodes in TimeLine。[#40700](https://github.com/ant-design/ant-design/pull/40700) [@any1024](https://github.com/any1024)
```
Comment (MadCcc) [inline CHANGELOG.zh-CN.md:23]: 这个中英文反了

File: CHANGELOG.zh-CN.md [hunk 6]
```diff
@@ -15,6 +15,21 @@ timeline: true
...
+- 🐞 修复 Silder 中的标记点 dot 可以正确触发 hover 和 click 的效果。[#40679](https://github.com/ant-design/ant-design/pull/40679) [@LongHaoo](https://github.com/LongHaoo)
+- 🐞 修复 Tour 不支持 `0` 作为节点的问题。[#40631](https://github.com/ant-design/ant-design/pull/40631) [@li-jia-nan](https://github.com/li-jia-nan)
+- 💄 修复 DataPicker.RangePicker 的 hover 范围样式。[#40607](https://github.com/ant-design/ant-design/pull/40607) [@Yuiai01](https://github.com/Yuiai01)
+- 💄 优化 Steps 组件自定义 `icon` 的大小。[#40672](https://github.com/ant-design/ant-design/pull/40672) [@MadCcc](https://github.com/MadCcc)
+- TypeScript
+  - 🤖 类型：更新 Upload 组件以支持泛型。[#40634](https://github.com/ant-design/ant-design/pull/40634) [@riyadelberkawy](https://github.com/riyadelberkawy)
```
Comment (MadCcc) [inline CHANGELOG.zh-CN.md:29]:
```suggestion
  - 🤖 Upload 组件支持泛型。[#40634](https://github.com/ant-design/ant-design/pull/40634) [@riyadelberkawy](https://github.com/riyadelberkawy)
```

File: CHANGELOG.zh-CN.md (hunk 5 above)
Comment (xrkffgg) [inline CHANGELOG.zh-CN.md:23]: 中文

File: CHANGELOG.en-US.md [hunk 7]
```diff
@@ -15,6 +15,21 @@ timeline: true
 
 ---
 
+## 5.2.1
+
+`2023-2-13`
+
+- 🛠 Rewrite `panelRender` function in Tour to funtion component。[#40670](https://github.com/ant-design/ant-design/pull/40670) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🐞 修复 TimeLine 中 `className` 传给子节点的行为。[#40700](https://github.com/ant-design/ant-design/pull/40700) [@any1024](https://github.com/any1024)
+- 🐞 Fix Silder dot can trigger click and hover correctly. [#40679](https://github.com/ant-design/ant-design/pull/40679) [@LongHaoo](https://github.com/LongHaoo)
```
Comment (MadCcc) [inline CHANGELOG.en-US.md:24]:
```suggestion
- 🐞 Fix Silder dot to trigger click and hover correctly. [#40679](https://github.com/ant-design/ant-design/pull/40679) [@LongHaoo](https://github.com/LongHaoo)
```

File: CHANGELOG.zh-CN.md [hunk 8]
```diff
@@ -15,6 +15,21 @@ timeline: true
 
 ---
 
+## 5.2.1
+
+`2023-2-13`
+
+- 🛠 重构 Tour 中 `panelRender` 函数为 FC。[#40670](https://github.com/ant-design/ant-design/pull/40670) [@li-jia-nan](https://github.com/li-jia-nan)
```
Comment (MadCcc) [inline CHANGELOG.zh-CN.md:22]:
```suggestion
- 🛠 重构 Tour 中 `panelRender` 为函数式组件。[#40670](https://github.com/ant-design/ant-design/pull/40670) [@li-jia-nan](https://github.com/li-jia-nan)
```
这个和下面的 Tour 合并吧

File: CHANGELOG.zh-CN.md [hunk 9]
```diff
@@ -15,6 +15,21 @@ timeline: true
 
 ---
 
+## 5.2.1
+
+`2023-2-13`
+
+- 🛠 重构 Tour 中 `panelRender` 为函数式组件。[#40670](https://github.com/ant-design/ant-design/pull/40670) [@li-jia-nan](https://github.com/li-jia-nan)
+- 🐞 修复 TimeLine 中 `className` 传给子节点的行为。[#40700](https://github.com/ant-design/ant-design/pull/40700) [@any1024](https://github.com/any1024)
```
Comment (MadCcc) [inline CHANGELOG.zh-CN.md:23]:
```suggestion
- 🐞 修复 TimeLine 中 `className` 传给子节点的问题。[#40700](https://github.com/ant-design/ant-design/pull/40700) [@any1024](https://github.com/any1024)
```

**Review by MadCcc — APPROVED** [review]

**Pull request merged**